assets/tests/components/notificationDrawer.test.tsx

```diff
@@ -433,3 +463,22 @@ const readNotification: Notification = {
   id: "1",
   state: "read" as NotificationState,
 }
+
+const routes: Route[] = [
```

Comment:
It seems weird to me to have this definition at the end of the file rather than near the beginning.

Reply:
Yeah, I thought this was weird as well but did it just because some of the other test data (`notification` and `readNotification`) was already defined down here. I can just move all of it to the top of the file, though.